app/templates/data-quality.hbs

```diff
@@ -40,17 +40,6 @@
             </p>
           </div>
         </Item>
-        <Item>
-          <div
-            class="au-u-flex au-u-flex--spaced-small au-u-flex--vertical-center"
-          >
-            <AuBadge @icon="dash" @skin="warning" @size="small" />
-            <p>
-              Er zijn documenten gevonden of meldingen ontvangen met dit
-              optioneel in te vullen attribuut
-            </p>
-          </div>
-        </Item>
```

Comment:
**Question:** Is it removed on purpose?

Reply:
It has not been implemented, so I removed it :)